src/debug/validatePreDebug.ts

```diff
@@ -144,27 +157,20 @@ async function validateWorkerRuntime(context: IActionContext, projectLanguage: s
     }
 }
 
-async function validateAzureWebJobsStorage(context: IActionContext, projectLanguage: string | undefined, projectLanguageModel: number | undefined, projectPath: string): Promise<void> {
-    if (canValidateAzureWebJobStorageOnDebug(projectLanguage)) {
-        const azureWebJobsStorage: string | undefined = await getLocalConnectionString(context, ConnectionKey.Storage, projectPath);
-        if (!azureWebJobsStorage) {
-            const functionFolders: string[] = await getFunctionFolders(context, projectPath);
-            const functions: ParsedFunctionJson[] = await Promise.all(functionFolders.map(async ff => {
-                const functionJsonPath: string = path.join(projectPath, ff, functionJsonFileName);
-                return new ParsedFunctionJson(await AzExtFsExtra.readJSON(functionJsonPath));
-            }));
-
-            // NOTE: Currently, Python V2+ requires storage to be configured, even for HTTP triggers.
-            if (functions.some(f => !f.isHttpTrigger) || pythonUtils.isV2Plus(projectLanguage, projectLanguageModel)) {
-                const wizardContext: IAzureWebJobsStorageWizardContext = Object.assign(context, { projectPath });
-                const wizard: AzureWizard<IAzureWebJobsStorageWizardContext> = new AzureWizard(wizardContext, {
-                    promptSteps: [new AzureWebJobsStoragePromptStep({ suppressSkipForNow: true })],
-                    executeSteps: [new AzureWebJobsStorageExecuteStep()]
-                });
-                await wizard.prompt();
-                await wizard.execute();
-            }
-        }
+async function validateAzureWebJobsStorage(context: IActionContext, projectLanguage: string | undefined, projectLanguageModel: number | undefined, projectPath: string, requiresDurableStorage: boolean): Promise<void> {
+    if (!(canValidateAzureWebJobStorageOnDebug(projectLanguage) || requiresDurableStorage)) {
```

Comment:
Ah, I see. Yeah, I understand the stylistic reasoning, though I actually feel the opposite. Unless there was a _lot_ of nesting, having a return in the beginning of a function definition feels ickier to me. But I know we do both in our code-base, so I'll leave it be.

I think `!condition_a || !condition_b` is quicker to parse than `!(condition_a || condition b)` though. Again, this might be a me problem though.

Reply:
> I think `!condition_a || !condition_b` is quicker to parse than `!(condition_a || condition b)` though.

Yeah agreed, I should have simplified that expression better, that way is probably far easier to parse quickly

Reply:
Ah yeah, you're right. Exactly why my brain was struggling so much lol

Reply:
I don't mind the guard clause, but can you change the conditional to not use distribution?